selfdrive/car/hyundai/hyundaican.py

```diff
@@ -29,7 +29,7 @@ def create_lkas11(packer, frame, car_fingerprint, apply_steer, steer_req,
     # FcwOpt_USM 2 = Green car + lanes
     # FcwOpt_USM 1 = White car + lanes
     # FcwOpt_USM 0 = No car + lanes
-    values["CF_Lkas_FcwOpt_USM"] = 2 if enabled else 1
+    values["CF_Lkas_FcwOpt_USM"] = 2 if enabled else 4
```

Comment:
Let's keep this the white icon. I'm pretty sure orange means something is faulted.

Reply:
I have changed back to white + lanes.